plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakePartitioning.java

```diff
@@ -140,11 +140,48 @@ public void testReadAllTypes()
     }
 
     @Test
-    public void testPartitionsSystemTableDoesNotExist()
+    public void testPartitionsSystemTableExists()
     {
-        assertQueryFails(
-                "SELECT * FROM \"partitions$partitions\"",
-                ".*'delta\\.tpch\\.\"partitions\\$partitions\"' does not exist");
+        assertQuerySucceeds("SELECT * FROM \"partitions$partitions\"");
```

Comment:
Add a corresponding test assertion in `TestDeltaLakeBasic#testCorruptedTableLocation`

Follow the work done in 1e188a17 to see whether you're missing anything.